graphstorm-processing/mypy.ini

```diff
@@ -0,0 +1,20 @@
+[mypy]
```

Comment:
What is this?

Reply:
Do we need this file?
Can we clean it?

Reply:
We use [mypy](https://mypy.readthedocs.io/en/stable/) and type annotations to statically check the correctness of our code.

However, many of the libraries we use do not support annotations, which makes mypy [trigger a large number of errors](https://mypy.readthedocs.io/en/stable/existing_code.html#fixing-errors-related-to-imports). This file fixes those errors.